testing/downloadSpark.sh

```diff
@@ -40,10 +40,28 @@ fi
 
 set -xe
 
-TIMEOUT_SEC=590
+MAX_DOWNLOAD_TIME_SEC=590
```

Comment:
Good catch, but I thought about keeping it for a while, reminding about great use of `timeout` that we have discarded for now. We possibly might what get back to it one day (i.e to reduce build time), once we get the build stability.